x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/UnresolvedAttribute.java

```diff
@@ -99,6 +99,11 @@ public String unresolvedMessage() {
         return unresolvedMsg;
     }
 
+    @Override
```

Comment:
A simple fix yet incorrect since an unresolved attribute should be first resolved before being manipulated.
It's way none of the `with` methods are implemented - touching an unresolved attribute is incorrect period.